src/main/resources/db/changelog/changes/vine-changelog-2023.02.28.xml

```diff
@@ -0,0 +1,110 @@
+    <changeSet id="1" author="volodymyr">
+        <createTable tableName="vine">
+            <column name="id" type="varchar(36)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="manufacturer" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="grape_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="color" type="varchar(36)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="is_sparkling" type="tinyint(4)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="sugar" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="country" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="region" type="varchar(255)"/>
+            <column name="year" type="int(4)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="price" type="float(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="amount" type="int(5)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="abv" type="float(5)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="post_office">
+            <column name="id" type="varchar(36)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="city" type="varchar(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="office_number" type="int(5)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="office_address" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="order">
+            <column name="id" type="varchar(36)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="user_id" type="varchar(36)">
+            </column>
+            <column name="user_email" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_first_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_last_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="phone_number" type="varchar(36)">
```

Comment:
are you sure that you need 36 symbols for phone_number storing?

Reply:
Just in case, again saw it in AC)
As an example "+380-67-11-14-900"
Would be valuable to change to varchar(17)

Reply:
i think it depend on the phone number format
i think it is possible even to store number like +380671114900 or even without +38 code if we store only Ukrainian mobile phone numbers